### PR TITLE
Custom consumer class

### PR DESCRIPTION
**Pull request by dvazar**
1. Added the ability to use a custom consumer class.
2. Ignore `visibility_timeout` if it is None, i.e. use the overall visibility timeout for the queue.

### CODE REVIEW

**Review by bidossessi — APPROVED** [review]
LGTM! Thanks.
Can you please rebase to fix the build issue?

File: dramatiq_azure/asq.py [hunk 1]
```diff
@@ -169,6 +169,10 @@ def __init__(
         self.queues: set = set()
         self.dead_letter = dead_letter
 
+    @property
+    def consumer_class(self):
```
Comment (bidossessi) [inline dramatiq_azure/asq.py:173]: Can you illustrate how you would use this?

Comment (dvazar) [reply dramatiq_azure/asq.py:186]: Sure. The main goal is to use a different `visibility_timeout` for each queue;

```python
class CustomBroker(ASQBroker):
    @property
    def consumer_class(self):
        return CustomASQConsumer


class CustomASQConsumer(ASQConsumer):
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        if self.queue_name == 'long':
            self.visibility_timeout = 20 * 60  # 20 minutes
        else:
            self.visibility_timeout = 5 * 60  # 5 minutes
```

**Review by bidossessi — DISMISSED** [review]
Please fix the test and I'll merge it.

**Review by bidossessi — APPROVED** [review]

**Pull request merged**